training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/spec.yaml

```
@@ -1,7 +1,7 @@
 $schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
 
 description: >-
-  An environment for deep learning with PyTorch containing the Azure ML SDK and additional python packages.
+  DEPRECATED: Please use pytorch-1.11-py38-cuda11.5-gpu instead. An environment for deep learning with PyTorch containing the Azure ML SDK and additional python packages.

```

I don't think it works well to mix the old description with the deprecation notice.
```suggestion
  DEPRECATED: Please use pytorch-1.11-cuda11.5 instead.
```